README.md

```diff
@@ -384,17 +384,17 @@ CDAClient cdaClient = clientBuilder.setCallFactory(httpClient).build();
 Proguard
 --------
 
-The [ProGuard configuration file](proguard-cda.cfg) is used to minify Android Apps using this SDK.
+The [ProGuard configuration file](proguard-cda.cfg) is used to minify Android Apps using this library.
 
 Rich Text renderer library
 --------
 
-There is [Java SDK for Rich Text API](https://github.com/contentful/rich-text-renderer-java). It helps in easily rendering rich text stored in Contentful using Java.
+There is [Java library for Rich Text API](https://github.com/contentful/rich-text-renderer-java). It helps in easily rendering rich text stored in Contentful using Java.
```

Comment:
Sure that that one should be renamed?
https://github.com/contentful/rich-text-renderer-java